packages/foam-vscode/package.json

```diff
@@ -342,6 +347,11 @@
           "type": "object",
           "description": "Custom graph styling settings. An example is present in the documentation.",
           "default": {}
+        },
+        "foam.selection.newNote.askForNoteTemplate": {
+          "description": "When creating a note from a text selection ask for template to be used",
+          "type": "boolean",
```

Comment:
generally speaking I am not a big fan of boolean options as they allow for little future flexibility (unless the option is clearly bound).
What do you think of a property with the following options? 

```
"foam.createNoteFromSelection.templateToUse" : {
          "type": "string",
          "default": "use new-note template",
          "enum": ['ask', 'use new-note template', 'use new-note-from-selection template'],
          ...
}
```

I am not feeling very strongly about the above, I am happy to explore the approaches here

Reply:
an interesting thing here is that we could use the same pattern with "create new notes from (wiki)links"